tests/test_s3/test_s3_eventbridge_integration.py

```diff
@@ -1,16 +1,23 @@
 import json
+from io import BytesIO
+from unittest import SkipTest
 from uuid import uuid4
 
 import boto3
 
-from moto import mock_aws
+from moto import mock_aws, settings
 from moto.core import DEFAULT_ACCOUNT_ID as ACCOUNT_ID
+from tests.test_s3.test_s3_multipart import reduced_min_part_size
 
 REGION_NAME = "us-east-1"
+REDUCED_PART_SIZE = 256
 
 
 @mock_aws
-def test_pub_object_notification():
+def test_put_object_notification_ObjectCreated_PUT():
+    if not settings.TEST_DECORATOR_MODE:
```

Comment:
What's the reason behind skipping the tests in ServerMode?

Reply:
Sorry, this part was added by mistake.